code/datums/craft/recipes/storage.dm

```diff
@@ -109,6 +109,15 @@
 	)
 	related_stats = list(STAT_COG)
 
+/datum/craft_recipe/lodge/leather_small_pouch
```

Comment:
```suggestion
/datum/craft_recipe/storage/leather_small_pouch
```

Reply:
also you need to remove the old one on the perk list

Reply:
oof, thanks

Reply:
> also you need to remove the old one on the perk list

Perk list?

Reply:
does this even complie now?